README.asciidoc

```diff
@@ -423,14 +398,16 @@ kakoune-lsp will find the most specific matching configuration to apply, where s
 
 Assuming the following configuration,
 
-[source,toml]
+[source,kak]
 ----
-[semantic_tokens]
-faces = [
-    {face="const_variable_declaration", token="variable", modifiers=["constant","declaration"]},
-    {face="const_variable", token="variable", modifiers=["constant"]},
-    {face="variable", token="variable"},
-]
+set-option global lsp_semantic_tokens %{
+    [semantic_tokens]
```

Comment:
You didn't have this header in the snippet above. It's probably fine if the daemon can accept both with minimal complexity, but examples should probably be consistent.

Reply:
Yeah the `[semantic_tokens]` header is gone; users shouldn't need to see it.
`faces = ` is still here; I tried making it just

```
[
   <face-spec1>,
   <face-spec2>,
]
```

but that on it's own is not valid TOML.

Perhaps we should make it a `str-list` option

Reply:
> that on it's own is not valid TOML

You could have the daemon add the `faces = ` before parsing, I guess.
But a `str-list` would also enable `-add` and `-remove`, so that's probably more useful anyways.

Reply:
Hmm might as well revive https://github.com/kakoune-lsp/kakoune-lsp/pull/558
It looks like this now:

```
set g lsp_semantic_tokens %{
    # face        token     modifiers...
    documentation comment   documentation
    comment       comment
    function      function
    keyword       keyword
    module        namespace
    operator      operator
    string        string
    type          type
    default+d     variable  readonly
    default+d     variable  constant
    variable      variable
}
```

Reply:
@topisani on discord pointed out that

> i think the token.modifier.modifier is the standard syntax for textmate scopes which the standard is based on, so the only thing is it would make it more in line with other systems

so how about we change the order, since the user already needs to do make manual changes to the config (once they chose to delete their `kak-lsp.toml`). I think putting the face first is weird.

Reply:
But putting it last would mean that you can't rely on its position. Since the amount modifiers is variable, putting a token after them could land at column 30 in one row, and at column 60 in the next.
You wouldn't be able to compare faces across lines anymore.

Reply:
I'd align them (using `&`) like the `lsp_semantic_tokens` above